frontend/src/components/LandingPage/LandingPage.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react'
+import MyNavbar from '../MyNavbar/MyNavbar';
 //Import components that you create here
 
 function LandingPage(){
     return(
         //Use components that you import here
-        <></>
+        <>
+             <MyNavbar/>
```

Comment:
Since thet navbar isnt going to be associated with just the landingpage, can we move it to App.js instead.

Reply:
Done!